otterdog/eclipse-xfsc.jsonnet

```
@@ -559,13 +559,14 @@ orgs.newOrg('technology.xfsc', 'eclipse-xfsc') {
         'dns',
       ],
     },
-    newXFSCRepo('train-trust-validator') {
+    newXFSCRepo('train-trusted-content-resolver') {
```

Renaming requires the use of "aliases"

```suggestion
    newXFSCRepo('train-trusted-content-resolver') {
      aliases: ['train-trust-validator'],
```

---

See https://otterdog.readthedocs.io/en/latest/userguide/renaming for details